platform-hub-api/app/controllers/me_controller.rb

```diff
@@ -40,6 +42,24 @@ def complete_hub_onboarding
     end
   end
 
+  def complete_services_onboarding
+    # Currently only supports GitHub onboarding
+    success = handle_onboard_github_request current_user, audit_context
+
+    current_user.update_flag :completed_services_onboarding, true
+    current_user.save  # Note: doesn't _really_ matter if this save fails – just means the flag won't get set
```

Comment:
Fair enough. I guess this flag then translates into something in the UI that user will be potentially presented / prompted with again?